lib/verify.js

```diff
@@ -16,15 +16,15 @@ module.exports = async (pluginConfig, { logger, cwd }) => {
     const ovsxPublishEnabled = isOvsxPublishEnabled();
     if (!vscePublishEnabled && !ovsxPublishEnabled) {
       throw new SemanticReleaseError(
-        'No personal access token was detected. Set the `VSCE_PAT` or the `OVSX_PAT` environment variable, at least one of them must be present when publish is enabled.\nLearn more at https://github.com/felipecrs/semantic-release-vsce#publishing',
+        'No personal access token was detected. Set the `VSCE_PAT`, `VSCE_USE_AZURE_CREDENTIALS`, or the `OVSX_PAT` environment variable, at least one of them must be present when publish is enabled.\nLearn more at https://github.com/felipecrs/semantic-release-vsce#publishing',
```

Comment:
I think VSCE_AZURE_CREDENTIALS would look better.

Also, please update the documentation (README).